doc/quick_start/contributor-guide.rst

```diff
@@ -0,0 +1,277 @@
+The mailing list is for announcements and questions for broad exposure and discussion. 
+JIRA is our issue tracking system, for feature requests and bug reports.
+
+.. _`Open Embedded guidelines for commit messages`: http://openembedded.org/wiki/Commit_Patch_Message_Guidelines
+.. _`Yocto Project`: http://yoctoproject.org
+.. _`Ostro OS Release Notes`: https://github.com/ostroproject/ostro-os/releases/
+.. _`Ostro-dev mailing list`: mailto://ostro-dev@lists.ostroproject.org
+.. _`ostroproject GitHub repo`: https://github.com/ostroproject/
+.. _`Ostro Project JIRA`: https://ostroproject.org/jira
+
+Prerequisites
+=============
+
+As a contributor, you'll want to be familiar with the Ostro OS project and the `Yocto Project`_ configuration and build tools.
+You should have set up your development environment (as introduced in our :ref:`quick_start`) 
+and successfully built an Ostro OS image (as introduced in :ref:`Building Images`). 
+As a Linux developer, you should also be familiar with common development tools such as Git and GitHub.
+
+Logging Bugs and Enhancement Requests
+=====================================
+
+Before starting on a patch, first check in `Ostro Project JIRA`_ to see what’s been reported on the issue you’re addressing. 
+Have a conversation on the mailing list to see what others think of your issue (and proposed solution). 
+You may find others that have encountered the issue you're finding, or with similar ideas for changes or additions. 
+If you don't, send a message to the mailing list to introduce and discuss your idea with the development community.
+
+After logging into JIRA, click “Issues” to search for open items. To avoid duplication, it's always a good practice 
+to search for existing or related issues before submitting your own. When you submit an issue (bug or feature request),
+the triage team will review the submission, typically within a few business days, and update the “Priority” 
+and “Target Version” and other program management fields. 
+
+
+Code Review Process Guidelines
+==============================
+
+Here are guidelines for the review process that you and the maintainers will follow when you submit a patch for the Ostro OS:
+
+#. Contributors follow the `Open Embedded guidelines for commit messages`_ and patches for recipes and other submissions.
+   In your commit message, briefly describe what the patch does, the problem it is addressing, or why it was submitted. 
+   Include additional detailed comments in the code itself, as appropriate.
+#. In the commit message, include the JIRA issue id (using this notation::
+
+    (Fixes|Relates-to) [OP-<number>]
+
+#. The general commit message format is::
+
+     <component>: <summary>
+     <description>
+     [Related-to|Fixes]: IOTOS-<number>
+     Signed-off-by: <name> <email>
+
+#. For each pull request (PR), the submitter and maintainer verify the patch does not cause a Continuous Integration (CI) build to break.
```

Comment:
"(re)test this please" is re-trigger. for approving testing from unknown sender should be used "ok to test".